src/python/pants/backend/python/target_types.py

```diff
@@ -1209,6 +1209,15 @@ class GenerateSetupField(TriBoolField):
     )
 
 
+class LongDescriptionPath(StringField):
+    """Path to a file that will be used to fill the long_description field in setup.py."""
```

Comment:
FYI that's not necessary. You already set it in `help`, which is all the Target API cares about